DigitalLearningSolutions.Data/Services/RegistrationService.cs

```diff
@@ -265,10 +246,19 @@ public void PromoteDelegateToAdmin(AdminRoles adminRoles, int? categoryId, int d
                 adminRoles.IsContentCreator,
                 adminRoles.IsCmsAdministrator,
                 adminRoles.IsCmsManager,
-                delegateUser.ProfileImage
+                delegateUser.JobGroupId
             );
 
-            var userId = userDataService.GetUserIdFromDelegateId(delegateId);
+            if (admin != null)
+            {
+                if (admin.Active)
+                {
+                    throw new AdminCreationFailedException(AdminCreationError.ActiveAdminAlreadyExists);
+                }
+
+                userDataService.UpdateAdminAccount(adminRegistrationModel, admin.Id);
+            }

```

Comment:
There are some incoming changes from the merge from mater around handling reactivating an inactive admin. We just need to make sure this behaves in the same way.